port_ocean/clients/port/mixins/integrations.py

```diff
@@ -99,15 +100,24 @@ async def patch_integration(
         handle_status_code(response)
         return response.json()["integration"]
 
+    def sirealize_logs(self,logs: list[dict[str, Any]]):
+        _logs = copy.deepcopy(logs)
+        for log in _logs:
+            if 'extra' in log.keys() and 'exc_info' in log['extra'].keys() and isinstance(log['extra']['exc_info'],Exception):
+                sirealized_exception = ''.join(format_exception(log['extra']['exc_info']))
+                log['extra']['exc_info'] = sirealized_exception
+        return _logs
```

Comment:
you are overriding the original `logs` object while you return `_logs`, is it a bug?